hackathon_site/event/serializers.py

```diff
@@ -85,6 +85,26 @@ def update(self, instance: Profile, validated_data):
         # ProfileSerializer update function and we want to keep the logic between these two update functions separate.
         return serializers.ModelSerializer.update(self, instance, validated_data)
 
+    def create(self, validated_data):
+        try:
+            self.context["request"].user.profile
+            raise serializers.ValidationError("User already has profile")
+        except:
+
+            response_data = {"attended": True}
```

Comment:
Just so you know, if this code did work, it would enter the except clause if an error is raised, which is when a user already has a profile so this logic doesn't make sense